test/specs/collections/Form/Form-test.js

```diff
@@ -44,21 +44,42 @@ describe('Form', () => {
 
   describe('onSubmit', () => {
     it('prevents default on the event when there is no action', () => {
-      const event = { preventDefault: sandbox.spy() }
+      const preventDefault = sandbox.spy()
 
-      shallow(<Form onSubmit={sandbox.spy()} />)
-        .simulate('submit', event)
-
-      event.preventDefault.should.have.been.calledOnce()
+      shallow(<Form onSubmit={_.noop} />)
+        .simulate('submit', { preventDefault })
+      preventDefault.should.have.been.calledOnce()
     })
 
     it('does not prevent default on the event when there is an action', () => {
-      const event = { preventDefault: sandbox.spy() }
+      const preventDefault = sandbox.spy()
 
-      shallow(<Form action='do not prevent default!' onSubmit={sandbox.spy()} />)
+      shallow(<Form action='do not prevent default!' onSubmit={_.noop} />)
         .simulate('submit', event)
+      preventDefault.should.not.have.been.called()
```

Comment:
Heads up, [`window.event`](https://developer.mozilla.org/en-US/docs/Web/API/Window/event) is a nonstandard proprietary IE global.  This test is not throwing because it is passing the `undefined` `window.event` to `simulate()`.

However, this test is also not testing anything since the event passed to simulate does not include the `preventDefault` spy under test.  I'll update these to assert on the actual event object again.